duckduckgo/iOS
# Remove Package.resolved and move to exact versioning

Task/Issue URL: https://app.asana.com/0/414235014887631/1202379485855918/f
Tech Design URL: 
CC:

**Description**:

This PR removes Package.resolved and updated the Xcode project to use exact versioning.

The versions used for exact versioning were taken from Package.resolved, hence why you'll see them changed in the diff. Compare the removed Package.resolved versions and verify that the new exact versions match the ones in that file.



**Steps to test this PR**:
1. Test that you can still build and run
2. Verify that the new exact package versioning match the versions in the removed Package.resolved file



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
